_meta/beat.reference.yml

```diff
@@ -21,14 +21,25 @@ apm-server:
   #ssl.certificate : "path/to/cert"
   #ssl.key : "path/to/private_key"
 
-  #frontend.enabled: false
-
-  # Rate Limit per second and IP address
-  #frontend.rate_limit: 10
-
-  # Comma separated list of permitted origins for frontend. User-agents will send
-  # a origin header that will be validated against this list.
-  # An origin is made of a protocol scheme, host and port, without the url path.
-  # Allowed origins in this setting can have * to match anything (eg.: http://*.example.com)
-  # If an item in the list is a single *, everything will be allowed
-  #frontend.allow_origins : *
+  #frontend:
```

Comment:
What do you think about adding a short message here that tells people that frontend support is experimental? It will be in the docs, but it would be good to document here as well.